src/runtime/providers/cloudinary.ts

```diff
@@ -73,7 +74,7 @@ const operationsGenerator = createOperationsGenerator({
     }
   },
   joinWith: ',',
-  formatter: (key, value) => `${key}_${value}`
+  formatter: (key, value) => key === 'e_blur' ? `${key}:${value}` : `${key}_${value}`
```

Comment:
I _feel_ dirty suggesting this. 😬 
I am not sure how this is supposed to be like as cloudinary has [a lot of transformations](https://cloudinary.com/documentation/transformation_reference#e_blur) with different formats e.g. `e_transformation:<value>` / `w_<value>` or without any value at all.
- should all be supported or is `e_blur` enough in this case?

Reply:
I would say that this is the way for doing it. Take a look at https://github.com/colbyfayock/cloudinary-util/blob/700863c02a18712e0ce01b8e7511449b21e40dc6/packages/url-loader/src/lib/transformations.ts#L35. This is currently the most supported cloudinary url loader and it uses the same approach. 

As there are other params like blur, I would recommend to changing it to something like this:

```suggestion
  formatter: (key, value) => key.includes('_') ? `${key}:${value}` : `${key}_${value}`
```